Control coverage/Feature/SubscriptionCore.md

```diff
@@ -979,3 +980,62 @@ By using Conditional Access policies for privileged roles, you can apply the rig
 
 ___ 
 
+## Azure_Subscription_AuthZ_Limit_Admin_Owner_Count 
+
+### Display Name 
+Minimize the number of admins/owners
+
+### Rationale 
+Each additional person in the admin/owner role increases the attack surface for the entire subscription. The number of members in these roles should be kept to as low as possible. 
+
+### Control Settings 
+```json 
+{
+    "ExcludeUsers": [],
+    "NoOfAdminOrOwnerLimit": 5,
+    "EligibleAdminOrOwnerRoles": [
+        "CoAdministrator",
+        "ServiceAdministrator",
+        "owner"
+    ]
+}
+ ```  
+
+### Control Spec 
+> **Passed:** 
+> The count of admin/owner accounts does not exceed 5.
```

Comment:
It would be better to say - The count of admin/owner accounts does not exceed the configured number of admin/owner count.